start_dev.sh

```
@@ -1,24 +1,44 @@
+
+# 1. Start Backend
 echo "Starting SolSniperX Backend..."
-cd /home/mulky/Desktop/SolSniperX/backend
+cd "$SCRIPT_DIR/backend"
+if [ ! -d "venv" ]; then
+    echo "Creating virtual environment and installing dependencies..."
+    python3 -m venv venv
+    ./venv/bin/pip install -r requirements.txt
+    ./venv/bin/python -m playwright install chromium
+fi
 source venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)